docs/content/guide/animations.ngdoc

```diff
@@ -294,42 +324,44 @@ app.config(function($animateProvider) {
 ```
 
 ```css
-/&#42; prefixed with animate- &#42;/
+/&#42; prefixed with `animate-` &#42;/
 .animate-fade-add.animate-fade-add-active {
   transition: all 1s linear;
   opacity: 0;
 }
 ```
 
-The classNameFilter approach generally applies the biggest speed boost, because the matching is
-done before any other animation disabling strategies are checked. However, that also means it is not
-possible to override class name matching with the two following strategies. It's of course still possible
-to enable / disable animations by changing an element's class name at runtime.
+The `classNameFilter` approach generally gives a big speed boost compared to other strategies,
+because the matching is done before other animation disabling strategies are checked. However, that
+also means it is not possible to override class name matching with the two following strategies.
+It's of course still possible to enable / disable animations by changing an element's class name at
+runtime.
 
 ### At runtime: {@link ng.$animate#enabled $animate.enabled()}
 
 This function can be used to enable / disable animations in two different ways:
 
-With a single `boolean` argument, it enables / disables animations globally: `$animate.enabled(false)`
-disables all animations in your app.
+With a single `boolean` argument, it enables / disables animations globally:
+`$animate.enabled(false)` disables all animations in your app.
 
 When the first argument is a native DOM or jqLite/jQuery element, the function enables / disables
 animations on this element *and all its children*: `$animate.enabled(myElement, false)`. This is the
 most flexible way to change the animation state. For example, even if you have used it to disable
```

Comment:
Is "This is the most flexible way to change the animation state." still true with the new customFilter? I don't think so, as $animate.enabled() does not work when the classNameFilter or customFilter are active. So let's remove this sentence